src/card/basic/card.directive.js

```diff
@@ -35,7 +149,32 @@ angular.module('patternfly.card').directive('pfCard', function () {
     scope: {
       headTitle: '@',
       subTitle: '@?',
-      showTopBorder: '@?'
+      showTopBorder: '@?',
+      showBottomBorder: '@?',
+      footer: '=?',
+      filter: '=?'
+    },
+    controller: function ($scope) {
+      if ($scope.filter && !$scope.currentFilter) {
+        if ($scope.filter.defaultFilter) {
+          $scope.currentFilter = $scope.filter.filters[$scope.filter.defaultFilter];
+        } else {
+          $scope.currentFilter = $scope.filter.filters[0];
+        }
+      }
+
+      $scope.footerCallBackFn = function () {
+        $scope.footerCallBackResult = $scope.footer.callBackFn();
```

Comment:
I know this won't get called if $scope.footer.callBackFn is not defined, but you might want to check just in case.